### DISCUSSION TITLE
pseudo windowrule doesn't work

### DISCUSSION THREAD

**Discussion by bobblkabb** [hyprwm/Hyprland#5359] [Q&A] [answered]
Hi,

what am I doing wrong?
These rules don't work:
```
windowrulev2 = pseudo, class:^([Tt]unar)$, title:^([Tt]hunar)$
windowrulev2 = size 900 900, class:^([Tt]unar)$, title:^([Tt]hunar)$
```

**Comment by vaxerski** [comment]
size is float-only, and ignored when tiled. It likely is pseudo, just of the size it's been launched at.

**Reply by bobblkabb** [reply]
So how can I make this work? Get it 'pseudoed' with a certain size.

**Reply by vaxerski** [reply, marked as answer]
you can't, atm.
